README.md

```diff
@@ -422,12 +422,41 @@ with at least one node that has persistent memory device(s). For development or
 testing, it is also possible to use a cluster that runs on QEMU virtual
 machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
 
+- **Make sure that the alpha feature gates CSINodeInfo and CSIDriverRegistry are enabled**
+
+The method to configure alpha feature gates may vary, depending on the Kubernetes deployment.
+
 - **Label the cluster nodes that provide persistent memory device(s)**
 
 ```sh
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Verify that the node labels have been configured correctly**
+
+```sh
+    $ kubectl get nodes --show-labels
+```
+
+Output of above command should indicate that every node with PMEM has these two labels:
+```
+pmem-csi.intel.com/node=<NODE-NAME>,storage=pmem
+```
+
+If **storage=pmem** is missing, label manually as described above. If **pmem-csi.intel.com/node** is missing, then double-check that the alpha feature gates are enabled and the CSI driver is running.
+
+- **Install addon storage CRDs if using Kubernetes 1.13 or 1.14**
+
+If you are not using the test cluster described in
+[Starting and stopping a test cluster](#starting-and-stopping-a-test-cluster)
+where CRDs are installed automatically, you must install those manually:
```

Comment:
Change "manually:" to "manually."  
Next line, change to "Note: This is a temporary solution that will be removed once the Kubernetes CSI sidecars for Kubernetes 1.14 are released."